README.md

```diff
@@ -27,9 +27,14 @@ group](https://github.com/domenic/infinite-list-study-group) research.
   const scroller = document.querySelector('virtual-scroller');
   const myItems = new Array(200).fill('item');
 
+  const clickHandler = (event) => {
+    console.log(`clicked item #${event.target.index}`);
+  };
+
   scroller.updateElement = (child, item, index) => {
     child.textContent = index + ' - ' + item;
-    child.onclick = () => console.log(`clicked item #${index}`);
+    child.index = index;
+    child.addEventListener('click', clickHandler);
```

Comment:
Hmm but see the discussion in #142. Basically if we're going to show using updateElement as the primary place to do updates, then we need to show factoring out the event listener like this. And I think the intro example only using updateElement, leaving element creation to the virtual-scroller itself, is probably better.

Reply:
I agree that factoring out event listeners like this is probably the right pattern to incentivize here. However, if this example should only use `updateElement`, then maybe an event listener isn't a good feature to use in it? Either your elements are all have identical 'shape' and using `createElement` is the appropriate place to add event listeners because they will never change and doing so in `updateElement` is redundant, or they are not and you would need a matching `removeEventListener` call in `recycleElement` to remove them because their shape may be changed later.